app/Http/Controllers/SchoolController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\School;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class SchoolController extends Controller
+{
+
+    public function uuid()
+    {
+        return Str::uuid()->toString();
+    }
+    
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request)
+    {
+        $id = $this->uuid();
+        $schoolName = $request->get('schoolName');
+        $schoolCode = $request->get('schoolCode');
+        $status = 'Active';
+        $school = new School();
+        $school->id = $id;
+        $school->name = $schoolName;
+        $school->code = $schoolCode;
+        $school->status = $status;
+
+        $checkSchool = School::where(['name' => $schoolName])->first();
+        if(empty($checkSchool)){
+            if($school->save()){
+                return response()->json(['success' => true], 200);
+            }
+            return response()->json(['success' => false], 200);
+        }
+        return response()->json(['error' => 'This school already exist'], 200);
```

Comment:
Same as previous comment, this response code should be 409 not 200. see https://stackoverflow.com/questions/3825990/http-response-code-for-post-when-resource-already-exists

Reply:
Noted

Reply:
Can you correct this and push so we merge the PR? Thank you.

Reply:
Thanks.